src/components/UNSTABLE__Shell/ProfilePopover/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file Profile popover.
+ * @copyright IBM Security 2020
+ */
+
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import ProfileImage from '../../ProfileImage';
+import PopoverHeader from '../PopoverHeader';
+
+import popoverNamespace from '../Popover';
+
+export const namespace = appendComponentNamespace(popoverNamespace, 'profile');
+const headerNamespace = appendComponentNamespace(namespace, 'header');
```

Comment:
Could this namespace be imported from `PopoverHeader`?

Reply:
Good catch!